### PR TITLE
added delete button to delete entries from table

### PR DESCRIPTION
**Pull request by sanatan-dive**
# Issue Resolved 24
- added delete button to delete entries from table
- rename dropdow-menu component to dropdown-menu

oie_jQ59Nq5aN4KQ

### CODE REVIEW

**Review by aayushchugh — CHANGES_REQUESTED** [review]
We need this feature for tax table too

File: src/components/ui/triple-dot-dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      strokeLinejoin="round"
+    >
+      <path d="M20 5H9l-7 7 7 7h11a2 2 0 0 0 2-2V7a2 2 0 0 0-2-2Z" />
+      <line x1="18" x2="12" y1="9" y2="15" />
+      <line x1="12" x2="18" y1="9" y2="15" />
+    </svg>
+  );
+}
+
+function FilePenIcon(props: SVGProps<SVGSVGElement>) {
```
Comment (aayushchugh) [inline src/components/ui/triple-dot-dropdown.tsx:54]: Please remove this unused function

File: src/components/ui/triple-dot-dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <DeleteIcon className="mr-2 h-4 w-4" />
+          Delete
+        </DropdownMenuItem>
+       
+      </DropdownMenuContent>
+    </DropdownMenu>
+  );
+}
+
+function DeleteIcon(props: SVGProps<SVGSVGElement>) {
```
Comment (aayushchugh) [inline src/components/ui/triple-dot-dropdown.tsx:33]: Instead of a backspace icon, we need a trash icon

**Review by aayushchugh — CHANGES_REQUESTED** [review]
Please create a new header named `actions` instead of directly showing a 3 dot icon

**Comment by sanatan-dive** [comment]
did the necessary changes
image

**Comment by sanatan-dive** [comment]
improved trash svg
image

**Review by aayushchugh — APPROVED** [review]
LGTM 🚀

**Pull request merged**